src/pages/proposal-commons/proposal-details/proposal-details.tsx

```diff
@@ -173,9 +172,9 @@ const ProposalDetailsContent = (props: ProposalDetailsProps) => {
         }
         content={
           <div className={styles.proposalDetailsSummary}>
-            <p className={classNames(styles.proposalDetailsItem, globalStyles.secondaryColor)}>
+            <pre className={classNames(styles.proposalDetailsItem, globalStyles.secondaryColor, styles.scrollX)}>
```

Comment:
Before:
![before](https://user-images.githubusercontent.com/22679154/129479160-88c47f9e-80b0-47e1-8392-ecfe60c043bb.png)

After:
![after](https://user-images.githubusercontent.com/22679154/129479159-9e68453e-bc9a-4616-ad3a-29c3ba540f42.png)

Reply:
If the content is too long, instead of wrapping let's add a scrollbar... This makes more sense, because people will be pasting URLs or IPFS links and for those it's better when they are not wrapped imo.
![scroll](https://user-images.githubusercontent.com/22679154/129479215-8db5d953-f641-4ebe-8fea-149560c8fe5b.png)

Reply:
If the content is too long, instead of wrapping let's add a scrollbar... This makes more sense, because people will be pasting URLs or IPFS links and for those it's better when they are not wrapped imo.
![scroll](https://user-images.githubusercontent.com/22679154/129479215-8db5d953-f641-4ebe-8fea-149560c8fe5b.png)